packages/slice-machine/src/stateDispatcher.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from "react";
+
+import { ServerState } from "@lib/models/server/ServerState";
+import { useDispatch } from "react-redux";
+import { getEnvironmentCreator } from "@src/modules/environment";
+
+type StateProviderProps = {
+  serverState: ServerState;
+};
+
+export const StateDispatcher = React.createContext({});
+
+const StateProvider: React.FunctionComponent<StateProviderProps> = ({
```

Comment:
same thing, it's basically just a useEffect to spread the serverState, it doesn't need to be a Provider

Reply:
Fixed